src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/DateTimeConverter.cs

```diff
@@ -24,7 +24,24 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
                 return null;
             }
 
-            var dateTime = DateTime.Parse(value.ToString());
+            DateTimeOffset dateTime;
+            if (value is DateTimeOffset dto)
+            {
+                dateTime = dto;
+            }
+            else if (value is DateTime dt)
+            {
+                dateTime = dt;
+            }
+            else
+            {
+                if (!DateTime.TryParse(value.ToString(), out dt))
+                {
+                    return null;
+                }
+                dateTime = dt;
+            }
```

Comment:
Please add tests for the 3 new code paths you introduced without tests:  [here](https://github.com/NuGet/NuGet.Client/pull/2542/files#diff-b0a55d2aaef5d51ad306107e7e8360c8R34), [here](https://github.com/NuGet/NuGet.Client/pull/2542/files#diff-b0a55d2aaef5d51ad306107e7e8360c8R40), and [here](https://github.com/NuGet/NuGet.Client/pull/2542/files#diff-b0a55d2aaef5d51ad306107e7e8360c8R42).